packages/react-workspace/react-client-app/package.json

```diff
@@ -25,5 +25,23 @@
     "sass-embedded": "^1.89.0",
     "typescript": "~5.4.5",
     "vite": "^7.0.0"
+  },
+  "optionalDependencies": {
```

Comment:
this looks a bit strange to do manually, is it possible to instead restore to an older lockfile from before the renovate merge until it's figured out?
(my hunch is that if it is related to vite changes this isn't the only project using vite so I'd wonder why it's the only one with the change)